lib/main.dart

```diff
@@ -20,7 +22,7 @@ class Concord extends StatelessWidget {
         title: 'Concord',
         // For testing, change the assignment of home to be whatever screen you're working on
         // Example: home: AuthScreen()
-        home: HomeScreen(),
+        home: RequestDetails(),

```

Comment:
```suggestion
        home: HomeScreen(),
```

I'd revert this change because it's not directly related to the changes of this PR.